modules/aws-emr-ports/output.tf

```diff
@@ -0,0 +1,21 @@
+locals {
+  applications  = [for app in var.applications : lower(app)]
+  running_hbase = contains(local.applications, "hbase")
+  running_spark = contains(local.applications, "spark")
+  output_hbase  = concat(var.master_ports_emr, var.master_ports_hbase, var.master_ports_ganglia, var.additional_ports)
```

Comment:
Ganglia is not required to be deployed on the cluster, so we want to treat it the hbase and spark treatment.

Reply:
Done